### ISSUE TITLE
issue of missing metadataBase and env variable

### ISSUE THREAD

**Issue by datacrud8** [LTopx/Le-AI#6]
I am trying to build this in local but build failed with issue of missing metadataBase.
when i check change log mentioned  that already done in 0.9.0
can you please tell me when it will be available V0.9.0?

Also please provide , what all are minimum environment variable need to run this app in local ?

Issue:
Server Error

TypeError: Invalid URL
This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source

src/app/layout.tsx (47:16) @ eval

  45 | },
  46 | appleWebApp: { capable: true, title: "Le-AI" },
> 47 | metadataBase: new URL(process.env.NEXT_PUBLIC_METABASE_URL as string),
     |              ^
  48 | openGraph: {

**Comment by Peek-A-Booo** [comment]
Sorry.

When I upgraded to a newer version of Next.js, I encountered a warning message: "_**metadata.metadataBase is not set for resolving social open graph or twitter images, using 'http://localhost:3000'"**_, so I added `metadataBase` in the metadata configuration.

But I forgot that I shouldn't add the required environment variables that would cause the packaging to fail.

Finally I have decided to remove the configuration of this variable when I checked the [Next.js metadataBase](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#metadatabase) again
